docs/src/main/tut/docs/effects/README.md

```diff
@@ -401,3 +401,125 @@ def programEmpty[F[_]: TraverseM] =
 
 programEmpty[TraverseM.Op].exec[List]
 ```
+
+## Validation
+
+The validation effect allows for the distinction between valid and invalid values in a program, accumulating the validation errors when executing it.
+
+The `validation` effect, like `state`, supports parametrization to any type remaining type safe throughout the program declaration.
```

Comment:
`state` here can be `[state](#state)` so when sbt-microsites https://github.com/47deg/sbt-microsites/issues/74 is fixed local anchor links will take you to the state definition it's referring to.